src/tests/trip/view-comments.test.js

```diff
@@ -0,0 +1,55 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import { loggedInToken, userWithNoTripToken } from '../fixtures/users.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('/Get view comments posted on trips requests', () => {
+  it('Should get comments posted on trip requests', (done) => {
+    chai.request(app)
+      .get('/api/trips/request/comments?page=1&limit=1')
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(200);
+        res.body.data.should.have.property('pageMeta');
+        res.body.data.should.have.property('rows');
+        res.body.data.rows.should.be.an('array');
+        res.body.data.rows[0].should.have.property('id');
+        res.body.data.rows[0].should.have.property('userId');
+        res.body.data.rows[0].should.have.property('subjectId');
+        res.body.data.rows[0].should.have.property('subjectType');
+        res.body.data.rows[0].should.have.property('comment');
+        res.body.data.rows[0].should.have.property('createdAt');
+        res.body.data.rows[0].should.have.property('updatedAt');
+        res.body.should.have.property('message').equal('List all comments');
+        done();
+      });
+  });
+
+  it('Should check for bad request', (done) => {
+    chai.request(app)
+      .get('/api/trips/request/comments?page=0&limit=2')
+      .set('Authorization', loggedInToken)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.status.should.be.equal(400);
+        res.body.should.have.property('message');
+        done();
+      });
+  });
+
+  it('Should not show requested trip comments when you did not create a comment', (done) => {
```

Comment:
@erickyvand Just because a user has not yet commented it does not mean an error has occurred. I would suggest using 204 as a response status.